CHANGELOG.yaml

```diff
@@ -12,6 +12,17 @@ meta:
 
 changes:
 
+  - title: Logger for ES request on /summary endpoint
```

Comment:
Remove this change log entry. The change log should only list changes that are observable by external users or that require an action by a team member when they upgrade their working copy to this version.

Reply:
@hannes-ucsc For the native test of change log, the failed test `test_changelog` specifies the following. 

```
    def test_changelog(self):
        self.run_setup('sdist', allow_fail=False)
        with open(os.path.join(self.package_dir, 'ChangeLog'), 'r') as f:
            body = f.read()
        # One commit, something should be in the ChangeLog list
        self.assertNotEqual(body, '')
``` 
And you specify the `logger` addition should be a different commit.